File: lista-de-exercicios/exercicio-4/Menu.ts
import readline from 'readline-sync';

export class Menu {
    private operador: string;
    private numero1: string;
    private numero2: string;

    public questoes() {
        this.operador = readline.question(
            'Escolha um operador para calcular:\n' +
            '+: para somar\n-: para subtrair\n*: para multiplicar\n/: para dividir\n^: para potencializar\n');
        this.numero1 = readline.question('digite o primeiro numero\n');
        this.numero2 = readline.question('digite o segundo numero\n');
    }

    public getOperador(): string {
        return this.operador;
    }

    public getNumero1(): number {
        return parseInt(this.numero1);
    }

    public getNumero2(): number {
        return parseInt(this.numero2); 
    }
}

File: lista-de-exercicios-java/src/main/java/org/example/exercicio04/Menu.java
package org.example.exercicio04;
import java.util.Scanner;

public class Menu {
    private char operador;
    private double numero1;
    private double numero2;

    public void questoes() {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Digite o primeiro número:");
        numero1 = scanner.nextDouble();
        System.out.println("Escolha um operador para calcular:");
        System.out.println("+: para somar");
        System.out.println("-: para subtrair");
        System.out.println("*: para multiplicar");
        System.out.println("/: para dividir");
        System.out.println("^: para potencializar");
        operador = scanner.next().charAt(0);
        System.out.println("Digite o segundo número:");
        numero2 = scanner.nextDouble();
    }

    public char getOperador() {
        return operador;
    }

    public double getNumero1() {
        return numero1;
    }

    public double getNumero2() {
        return numero2;
    }
}
